src/redisearch_api.c

```diff
@@ -55,6 +55,24 @@ void RediSearch_DropIndex(IndexSpec* sp) {
   RWLOCK_RELEASE();
 }
 
+char **RediSearch_IndexGetStopwords(IndexSpec* sp) {
+  return GetStopWordsList(sp->stopwords);
+}
+
+double RediSearch_IndexGetScore(IndexSpec* sp) {
+  if (sp->rule) {
```

Comment:
Does the user can even set default score from llapi? If no then its irrelevant to expose it no? I think that sp->rule are never set when used from llapi (we do not follow hashes), am I right?

Reply:
You are right.
@swilly22 WDYT? should we return to you the default which you can't change? or would you just put the default yourself?

Reply:
Why not add an API to set it when you create the index?

Reply:
score and language added to `RSIndexOptions` and can be set.

Reply:
So why not allow setting them on the index option and then use it?